files/upload_repo.bash

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
```

Comment:
I'd actually prefer not to make this change now. The script has specific exit code handling for the ssh command from upstream. I can move it into a separate PR though.

Reply:
The `-e` should prevent us for possible failures in the script and return a success exit code. We can move to a different PR, it can affect heavily the behavior if some minor problems exist.

Reply:
I fully agree that errexit / -e is better for any bash script. My concern is that if we *rely* on the script working during a failure right now and don't know it I don't want to change that during the migration window. Hence moving this out to anther PR.

Reply:
Branch is up to be rebased for a future PR. https://github.com/ros-infrastructure/cookbook-ros-buildfarm/tree/upload-script-errexit